src/native/managed/cdacreader/tests/LoaderTests.cs

```diff
@@ -19,21 +21,19 @@ public void GetPath(MockTarget.Architecture arch)
         TargetTestHelpers helpers = new(arch);
         MockMemorySpace.Builder builder = new(helpers);
         MockLoader loader = new(builder);
```

Comment:
By switching from creating a "real" `ContractDescriptorTarget` to a `TestPlaceholderTarget`, we are bypassing the `ContractDescriptorParser`. It looks like we already test the contractdescriptor reading in other places, is the benefit that it is easier to mock out contracts?